### DISCUSSION TITLE
Invoke external webhook when there are health issues and/or `/health` url listener to return HTTP 500 when there are health problems

### DISCUSSION THREAD

**Discussion by ramonsmits** [teslamate-org/teslamate#1293] [Ideas]
**Describe the bug**

For some unknown reason, TeslaMate was not able to retrieve car data. I restarted the container which resolved the problem but now I have a 2-day gap in my data. I wish I 

On the dashboard it showed "Healthcheck failed" but I wish I was somehow notified by the failing check.

For example, can a query be done at `/health` or `/healthcheck` or to invoke a webhook to a service like PushOver to signal any health issues.

The original health issue was related to these reoccurring logs:

![image](https://user-images.githubusercontent.com/152998/106903965-eaad1180-66fa-11eb-8913-487811784327.png)

For example, in release 2.21.2 the token will expire. The health check could return 500 in advance, knowing that the token requires a refresh.

**Environment**

- TeslaMate version:
- Type of installation: Docker
- OS TeslaMate is installed on:
- User OS & Browser:
- Others:

**Comment by adriankumpf** [comment]
> For example, in release 2.21.2 the token will expire. The health check could return 500 in advance, knowing that the token requires a refresh.

In this example, restarting the container wouldn't achieve anything, right? The "Healthcheck failed" message refers solely to the fact that communication with the Tesla servers is not possible. Be it because your server is offline or because there are problems with the Tesla API. If you want to be notified immediately when this is the case, MQTT could perhaps be an option (topic:`teslamate/cars/$car_id/healthy`)?
